spec/support/controller_helper.rb

```diff
@@ -118,7 +118,8 @@ def report_data_request_data(options)
       # Fires a POST request to the current controller's /report_data action
       #
       def report_data_request(options)
-        post :report_data, :params => report_data_request_data(options)
+        request.headers['Content-Type'] = 'application/json'
+        post :report_data, report_data_request_data(options)
```

Comment:
The lines above are required, otherwise the `report_data` post request is broken. Rails can't handle properly nested arrays on request. This is the way to bypass it.

If `:named_scope` is sent the old way this is what happens:

Sent data (includes):
```ruby
"named_scope" => [["with_requester", 80000000000202]]
```
Received by controller:
```ruby
"named_scope" => [["with_requester"], [80000000000202]]
```